xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -825,6 +825,34 @@ struct DatabaseService {
             )
         }
     }
+    
+    func readRandomEntryInDateRange(startDate: Date, endDate: Date) -> EntryStub? {
+        try? database.execute(
+            sql: """
+            SELECT slug, body
+            FROM entry
+            WHERE entry.modified BETWEEN date(?) AND date(?)
```

Comment:
I tried a few different ways of achieving this, but this felt the most generally flexible for future use.